packages/components/copybutton/src/CopyButton.styles.ts

```diff
@@ -1,12 +1,27 @@
 import { css } from 'emotion';
 import tokens from '@contentful/f36-tokens';
+import type { CSSObject } from '@emotion/serialize';
 
-export const getStyles = () => ({
+const sizeToStyles = ({ size }): CSSObject => {
+  switch (size) {
+    case 'small':
+      return {
+        height: `32px`,
+        width: `32px`,
+      };
+    default:
+      return {
+        height: `40px`,
+        width: `40px`,
+      };
+  }
+};
+
+export const getStyles = ({ size }) => ({
```

Comment:
Btw, since it's only two options

You could move that logic here and simplify it, like this:

```ts
export const getStyles = ({ size }) => {
const sizeValue =  size === 'small' ? '32px' : '40px';

return ({
  wrapper: {
    // ...
    width: sizeValue
    height: sizeValue
  },
  // all the styles you have already
})
```